massa-node/src/main.rs

```diff
@@ -135,10 +136,15 @@ async fn launch() -> (
     let ledger_config = LedgerConfig {
         initial_sce_ledger_path: SETTINGS.ledger.initial_sce_ledger_path.clone(),
     };
+    let async_pool_config = AsyncPoolConfig {
+        // TODO: define max_length value, it musn't come from a settings file
+        max_length: 100,
```

Comment:
Why not defining in default.rs like the others values ? Is default.rs considered as a config file?